docs/tutorial/notebook/03_train_model.ipynb

```diff
@@ -0,0 +1,583 @@
+{
```

Comment:
<p>Both PyTorch and TF can be installed. User code just needs to choose one inside a specific python file.</p><p><code>requirements-torch.txt </code>if Nvidia GPU is not available. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/isl-org/Open3D-ML/pull/545/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>